nuxeo-core/nuxeo-core-test/pom.xml

```diff
@@ -192,6 +192,11 @@
       <artifactId>commons-lang3</artifactId>
     </dependency>
 
+    <dependency>
+      <groupId>org.jsoup</groupId>
+      <artifactId>jsoup</artifactId>
+    </dependency>
+
```

Comment:
as discussed we should remove this dependency or i miss something :(. the idea is to keep it in the pom parent as a dependency management and we will added the to specific pom when needed for example in the comment platform project as scope `test` dependency